src/app/Http/Controllers/UserInterfaceController.php

```diff
@@ -42,8 +38,8 @@ public static function handle(Request $request): Response
             'isLanguageSelectorEnabled' => legacy_config('enable_language_selector'),
             'languageMapping' => self::getLanguageMapping(),
             'formatLangNames' => legacy_config('format_lang_names', []),
-            'meetingStatesAndProvinces' => implode(',', legacy_config('meeting_states_and_provinces', [])),
-            'meetingCountiesAndSubProvinces' => implode(',', legacy_config('meeting_counties_and_sub_provinces', [])),
+            'meetingStatesAndProvinces' => legacy_config('meeting_states_and_provinces', []),
+            'meetingCountiesAndSubProvinces' => legacy_config('meeting_counties_and_sub_provinces', []),
```

Comment:
Do any clients depend on this being a comma delimited list? Does BMLT Workflow use this?